### PR TITLE
Employee endpoints

### PR DESCRIPTION
**Pull request by Minhly**

### CODE REVIEW

File: API/Controllers/EmployeesController.cs [hunk 1]
```diff
@@ -72,7 +72,66 @@ public async Task<IActionResult> GetEmployeesWithWarehouseInfoG()
                 return BadRequest(ModelState);
             }
 
-            return Ok(employees);
+            var employeeDto = new List<EmployeeWarehouseDto>();
+
+            foreach (var a in employees)
```
Comment (Anders164a) [inline API/Controllers/EmployeesController.cs:77]: Generel naming, maybe use something other than "a" through the project?

File: API/Service/Repositories/EmployeeRepository.cs [hunk 2]
```diff
@@ -17,13 +17,28 @@ public EmployeeRepository(surstroemContext dbcontext)
...
-                .Include(a => a.Id)
-                .Include(c => c.Warehouse)
-                .Include(s => s.Warehouse.WarehouseType.Type)
-                .ToListAsync();
+                    .Include(q => q.User)
+                    .Include(c => c.Warehouse)
+                    .Include(s => s.Warehouse.WarehouseType)
+                    .Include(f => f.Warehouse.Address)
+                    .Include(a => a.Warehouse.Address.PostalCode)
+                    .Include(w => w.Warehouse.Address.PostalCode.Country)
```
Comment (Anders164a) [inline API/Service/Repositories/EmployeeRepository.cs:40]: Wrote a review to this in the wrong branch but...

Where does q, c, s, etc. come from, and where does it know what it is?

Could it be named differently?

**Review by Anders164a — CHANGES_REQUESTED** [review]
See comments

File: API/Controllers/EmployeesController.cs (hunk 1 above)
Comment (Minhly) [reply API/Controllers/EmployeesController.cs]: Done.

**Review by Anders164a — APPROVED** [review]
🚀

**Pull request merged**